app/models/game.rb

```diff
@@ -1,10 +1,42 @@
 class Game < ApplicationRecord
   attr_accessor :messages
 
-  enum current_turn: ["challenger", "computer"]
+  enum current_turn: ['player_1', 'player_2']
   serialize :player_1_board
   serialize :player_2_board
 
   validates :player_1_board, presence: true
   validates :player_2_board, presence: true
+
+  def active_player
+    if current_turn == 'player_1'
```

Comment:
@anon0mys Could we just return `current_turn`?

Reply:
Oh wait, nvm. Missed that it was a string. Try this...

`self.send(current_turn)`